bcda/cli.go

```diff
@@ -120,17 +128,19 @@ func importCCLF0(fileMetadata *cclfFileMetadata) (map[string]cclfFileValidator,
 	}
 
 	if _, ok := validator["CCLF8"]; !ok {
+		fmt.Printf("failed to parse CCLF8 from CCLF0 file: %v", fileMetadata)
 		err := fmt.Errorf("failed to parse CCLF8 from CCLF0 file: %v", fileMetadata)
 		log.Error(err)
 		return nil, err
 	}
 	if _, ok := validator["CCLF9"]; !ok {
+		fmt.Printf("failed to parse CCLF9 from CCLF0 file: %v", fileMetadata)
 		err := fmt.Errorf("failed to parse CCLF9 from CCLF0 file: %v", fileMetadata)
 		log.Error(err)
 		return nil, err
 	}
-	fmt.Printf("Imported CCLF0 file %s.\n", fileMetadata.name)
-	log.Infof("Imported CCLF0 file %s.", fileMetadata.name)
+	fmt.Printf("Successfully imported CCLF0 file %s.\n", fileMetadata.name)
+	log.Infof("Successfully imported CCLF0 file %s.", fileMetadata.name)
```

Comment:
Could it be unsuccessfully imported? Seems like this change from the old message might not be necessary.

Reply:
it's just more descriptive in my opinion especially since earlier in that method we have "Importing CCLF%d file".(Importing vs. imported) is now (Importing vs. Successfully imported)